frontend/src/components/Modal/RiderModal.tsx

```diff
@@ -40,7 +47,20 @@ const RiderModal = ({ existingRider, isRiderWeb }: RiderModalProps) => {
     setIsSubmitted(true);
     closeModal();
   };
-
+  useEffect(() => {
+    if (deleteStudent) {
+      fetch(
+        `/api/riders/${!existingRider ? '' : existingRider.id}`,
+        withDefaults({
+          method: 'DELETE',
+        })
+      ).then(() => {
+        setToast(true);
+        refreshRiders;
```

Comment:
here's the error, you're missing the parentheses to call the refreshRiders function

but we can actually make this simpler without using the `useEffect` and just using `studentDelete`:
```
  const studentDelete = () => {
    fetch(
      `/api/riders/${!existingRider ? '' : existingRider.id}`,
      withDefaults({
        method: 'DELETE',
      })
    )
      // you can use refreshRiders without () here, since we're passing it as a function
      .then(refreshRiders)
      .then(() => {
        setToast(true);
        history.push('/riders');
      });
  };
```

this avoids unnecessary rerenders from updating `deleteStudent`